src/components/anchor-navigation/anchor-navigation.spec.tsx

```diff
@@ -145,7 +145,7 @@ describe("AnchorNavigation", () => {
 
       act(() => {
         window.dispatchEvent(new Event("scroll"));
-        jest.advanceTimersByTime(150);
+        jest.advanceTimersByTime(40);
```

Comment:
Why was this update?

Reply:
I'm not sure why but with the previous value after updating the jest I was not getting coverage for this line of code
https://github.com/Sage/carbon/blob/master/src/components/anchor-navigation/anchor-navigation.component.tsx#L133

Reply:
No problem, we could maybe just use `runAllTimers` instead of `advanceTimersByTime` but it's not a big deal so I'm happy to just ✅